cart/src/App.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from "react";
+import Cart from "./components/Cart";
+import Home from "./components/Home";
+import NavBar from "./components/Navbar";
+
+const App = () => {
+  const [showScreen, setShowScreen] = useState(true);
+  const [cartArray, setCartArray] = useState([]);
+  const [totalPrice, setTotalPrice] = useState(0);
+
+  const addItemToCart = (items) => {
+   if (!isItemExits(items)) {
+      setCartArray([...cartArray, items]);
+    }
+  };
+
+  const isItemExits = (items) => {
+    const isPresent = cartArray.find((item) => item.id === items.id);
+
+    if (isPresent) {
+      alert("Item is already added to cart");
+    } else {
+      alert("Added to cart");
+    }
+
+    return isPresent;
+  };
+
+  const handleIncDec = (operator, items) => {
```

Comment:
function name is also not clear, should it be `updateItemQuantity`

again - `items` should not be the variable name here as well